dll/win32/shell32/CNewMenu.cpp

```diff
@@ -772,6 +772,6 @@ CNewMenu::Initialize(PCIDLIST_ABSOLUTE pidlFolder,
 
     /* Load folder and shortcut icons */
     m_hIconFolder = (HICON)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDI_SHELL_FOLDER), IMAGE_ICON, 16, 16, LR_SHARED);
-    m_hIconLink = (HICON)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDI_SHELL_SHORTCUT), IMAGE_ICON, 16, 16, LR_SHARED);
+    m_hIconLink = (HICON)LoadImage(shell32_hInstance, MAKEINTRESOURCE(IDI_SHELL_SHORTCUT - 1), IMAGE_ICON, 16, 16, LR_SHARED);
```

Comment:
This is not correct change... this way it will load preceding icon instead. And this LoadImage is bound to shell32.

Reply:
I think it would be better to get new implemented functions in #6421, then it will be possible to address these problems by using them.

Reply:
> This is not correct change... this way it will load preceding icon instead. And this LoadImage is bound to shell32.

Oops, indeed. Just noticed my mistake.

> I think it would be better to get new implemented functions in #6421, then it will be possible to address these problems by using them.

OK, maybe this is indeed better.

Reply:
OK, just improved #6421. I think now I can just use `SIC_LoadOverlayIcon` here to load the current icon from registry instead of specifying default resource ID.

Reply:
Not sure using `SIC_LoadOverlayIcon` would be correct here (it returns an `int`), but yeah, let's handle that PR first.